FableAnniversary.asl

```diff
@@ -2,6 +2,9 @@ state("Fable Anniversary") {
 	int questsCompleted : 0x322FD00, 0x6C, 0x44, 0x14, 0xc4;
 	int isLoading : 0x322139C, 0x1DC, 0x130;
 	int isLoadingSave: "Fable Anniversary.exe", 0x3230374, 0x08, 0x104;
+	int isStarting: "Fable Anniversary.exe", 0x3232770;

```

Comment:
Is this actually `int` or is it `bool`?

Reply:
Also, can you explain what this is please

Reply:
It's a bool, but I wrote it as an int so that I could check if it was going from 0 to 1 so that if you were still, you could reset, and the timer wouldn't restart again, and it's a static pointer for cutscenes and movies

Reply:
Could you put it as a bool again please if that's its type